src/server/helpers/diffFormatters/entity.js

```diff
@@ -236,6 +236,14 @@ function formatIdentifier(change) {
 	return null;
 }
 
+function formatRelationshipAttrubuteAddOrDelete(relationshipAttributes) {
```

Comment:
> I think this may be because in the formatRelationship function we detect addition and removal of relationships, but not modifications?

Yes, you are right. For now, I haven't added the functionality for detecting modification. If you remember the issue I reported earlier regarding the diffs of modified relationships, that could be fixed after we replace our diffing libraray.

Reply:
I do remember the issue, I suppose I was expecting to see the diff even if it has some issues. I think that would be preferable to having an empty diff view like https://test.bookbrainz.org/revision/29907

Reply:
cool, will add it then :+1: